assets/product-images-slider.js

```diff
@@ -0,0 +1,62 @@
+const thumbnailSlides = document.querySelector('[data-glide-thumbnail-slides]').dataset.glideThumbnailSlides
+const thumbnailsPerView = thumbnailSlides ? thumbnailSlides : 0
+
+const glideOptions = {
+  image: {
+    type: 'carousel',
+    startAt: 0,
+    perView: 1,
+    focusAt: 'center',
+    gap: 0,
+    dragThreshold: 10,
+    peek: 40,
+  },
+  thumbnail: {
+    type: 'carousel',
+    startAt: 0,
+    perView: thumbnailsPerView,
+    focusAt: thumbnailsPerView % 2 === 0 ? (thumbnailsPerView / 2) - 1 : 'center'
+  }
+}
+
+const createGlideInstance = (element, optionsObject) => {
+  return new Glide(element, optionsObject)
+}
+
+const mountGlideInstance = (instance) => {
+  instance.mount()
+}
+
+const onSlideOfProductImage = (instance, staticInstance) => {
+  instance.on(['mount.after', 'run'], function() {
```

Comment:
I'm guessing this is functiohn() because it was copied from a source online? This could be an arrow function and it would be more consistent

Reply:
Yes, this is from the Glide JS documentation.